valhalla/mjolnir/node_expander.h

```diff
@@ -52,12 +52,14 @@ struct Edge {
                                //   short enough it may be internal to
                                //   an intersection
     uint64_t drivable_ferry : 1;
-    uint64_t reclass_ferry : 1; // Has edge been reclassified due to
-                                // ferry connection, will remove dest_only tag from DE
-    uint64_t turn_channel : 1;  // Link edge should be a turn channel
-    uint64_t way_begin : 1;     // True if first edge of way
-    uint64_t way_end : 1;       // True if last edge of way
-    uint64_t spare : 25;
+    uint64_t reclass_ferry : 1;        // Has edge been reclassified due to
+                                       // ferry connection, will remove dest_only tag from DE
+    uint64_t turn_channel : 1;         // Link edge should be a turn channel
+    uint64_t way_begin : 1;            // True if first edge of way
+    uint64_t way_end : 1;              // True if last edge of way
+    uint64_t importance_hierarchy : 4; // will be set when edge needs to be moved in the hierarchy
```

Comment:
3 would be enough for road class stuff. I added 4 bits so I can default this to 8 (`kInvalidRoadClass`).